src/routes/styleguide/__layout.reset.svelte

```diff
@@ -0,0 +1,25 @@
+<script context="module" lang="ts">
+  import { mode } from '$app/env';
+
+  /** @type {import('@sveltejs/kit').Load} */
+  export async function load({}) {
+    let responseObj: any = {
+      status: 200,
+    };
+
+    if (mode !== 'development') {
+      responseObj = {
+        status: 404,
+        error: new Error('Could not find page'),
```

Comment:
I'm pretty sure that this will actually throw an exception in this function and blow up (resulting in a 500 with a stack trace and not a 404). You probably just want to set the error property on the object as a string.

Reply:
https://kit.svelte.dev/docs#loading

That's how they do the error message in the example and it seems to work. 

Throwing an error here results in an error page bubble up to the root error page (which we don't have) which eventually will throw a 500. I tried hitting some random routes and they hold onto the connection until they 500 out so we probably need to look into that eventually.